transport-native-epoll/pom.xml

```diff
@@ -179,7 +179,7 @@
                       <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                     </manifest>
                     <manifestEntries>
-                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_epoll.so; osname=linux; processor=x86_64,*</Bundle-NativeCode>
+                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_epoll_${os.detected.arch}.so; osname=linux; processor=${os.detected.arch},*</Bundle-NativeCode>
```

Comment:
an alternative option is, could remove <platform>.</platform> hawtjni will then place these in folders correctly based on the arch

Reply:
this would have been more complex as we would also need to adjust our code that loads the native libs etc.

Reply:
Fair enough, as noted this approach works +1